.travis.yml

```diff
@@ -3,7 +3,6 @@ sudo: false
 fast_finish: true
 python:
   - 2.7
-  - 3.4
```

Comment:
Should we still run tests for Py34?

Reply:
I was seeing build failures from missing dependencies in py34, but we're no longer running anything on py34, so it's ok to remove.